### ISSUE TITLE
Xbox Series X controllers not working with retroarch

### ISSUE THREAD

**Issue by GCRVL** [ChimeraOS/chimeraos#897]
My Xbox Series X controllers work well in ChimeraOs, but when booting any rom the popup "Xbox 360 gamepad not configured" pops up and dissallows me to control any retroarch game.

Disabling Steam Input in the controller menu solves the issue, then the gamepad is recognized in Retroarch.

RetroArch should be configured to handle Steam Input, or Steam Input should be disabled for all Retroarch games I believe.

**Comment by alkazar** [comment]
Strange. I cannot reproduce. The virtual ateam controller does not interfere.

I would like to switch to using Steam Input, but when I tried that the physical controller is still recognized by Retroarch and I experience what you describe in this ticket.

Did you perhaps manually configure your controller in Retroarch?

**Comment by GCRVL** [comment]
I don't even know how to get to the Retroarch configuration. 😄 

Fresh install on bare metal, Uploaded ROMs via app (GBA, N64), "Controller not configured" for Bluetooth connected Xbox Series X controllers. Works when disabling Steam Input.

I am happy with the workaround but it's not ideal.

**Comment by pastaq** [comment]
I don't think we can disable steam input by default for just retroarch unfortunately. I'll brainstorm on a solution though.

**Comment by GCRVL** [comment]
Tried grabbing a screenshot but the upload screenshot to Steam Cloud function does not work.

Exact error is:

"""Microsoft X-Box 360 pad 0 (10462/4607) not configured

Xbox One S Wireless Controller configured in port 2"""

**Comment by GCRVL** [comment]
And some dmesg logs:

```
[    7.653978] input: Xbox Wireless Controller as /devices/virtual/misc/uhid/0005:045E:0B13.0009/input/input31
[    7.654128] hid-generic 0005:045E:0B13.0009: input,hidraw6: BLUETOOTH HID v5.09 Gamepad [Xbox Wireless Controller] on 14:ac:60:55:12:8a
[    7.770533] input: Xbox Wireless Controller as /devices/virtual/misc/uhid/0005:045E:0B13.0009/input/input32
[    7.770616] microsoft 0005:045E:0B13.0009: input,hidraw6: BLUETOOTH HID v5.09 Gamepad [Xbox Wireless Controller] on 14:ac:60:55:12:8a
[    7.857336] input: Xbox Wireless Controller as /devices/virtual/misc/uhid/0005:045E:0B13.000A/input/input33
[    7.857482] microsoft 0005:045E:0B13.000A: input,hidraw7: BLUETOOTH HID v5.09 Gamepad [Xbox Wireless Controller] on 14:ac:60:55:12:8a
```

**Comment by alkazar** [comment]
> I don't think we can disable steam input by default for just retroarch unfortunately. I'll brainstorm on a solution though.

We can, but I would like to avoid it. The ideal solution that I am working towards would be to use Steam Input only.

**Comment by alkazar** [comment]
Do you have any other peripherals connected? Try disconnecting everything.

For example, in my case, I had a steering wheel that was interfering and causing the issue you describe.

**Comment by koblez** [comment]
I have a similar problem on Linux with the flatpak version of RetroArch on version 1.22.2 on two different systems (one with Linux Mint, the other with SteamOS).
But instead of a X-Box Controller I have a Steam Controller (2026).
When I launch any ROM or RetroArch itself trough Steam I also get the error `Microsoft X-Box 360 pad 0 (10462/4607) not configured`.
The Steam Controller will be bind to that pad and it's buttons will mostly be usable (back paddles don't work).
I can't save the configuration in a controller profile with the error `Error saving controller profile`
